modules/schematics/src/reducer/files/__name@dasherize@if-flat__/__name@dasherize__.reducer.ts.template

```diff
@@ -1,7 +1,9 @@
-import { Action, createReducer, on } from '@ngrx/store';
+import { Action,<% if(feature) { %> createFeature,<% } %> createReducer, on } from '@ngrx/store';
 <% if(feature) { %>import * as <%= classify(name) %>Actions from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
```

Comment:
Unrelated to your change, but the current reducer creates an invalid file.
This is because `FooActions` is used, but not imported.
As a fix we can just import the actions.

```suggestion
import * as <%= classify(name) %>Actions from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';
```